code/modules/cargo/exports/weapons.dm

```diff
@@ -69,4 +69,37 @@
 	cost = 3
 	unit_name = "pair"
 	message = "of handcuffs"
-	export_types = list(/obj/item/restraints/handcuffs)
\ No newline at end of file
+	export_types = list(/obj/item/restraints/handcuffs)
+	
+// relics of lavaland
+
+/datum/export/weapon/hierophant
+	cost = 50000
+	unit_name = "Hierophant Club"
+	export_types = list(/obj/item/hierophant_club)
+	
+/datum/export/weapon/lava
+	cost = 45000
+	unit_name = "Lava Staff"
+	export_types = list(/obj/item/lava_staff)
+	
+/datum/export/weapon/cleaving_saw
+	cost = 40000
+	unit_name = "Cleaving Saw"
+	export_types = list(/obj/item/melee/transforming/cleaving_saw)
+	
+/datum/export/weapon/immortality_talisman
+	cost = 25000
```

Comment:
This one is a random drop and the others are megafauna drops.

Reply:
the plan at the end is to add every item to it megafauna being high priced as normal tendril loot a little lower

Reply:
if it's not done in the PR you shouldn't have stuff like this set preemptively. It should be the equivalent to a normal drop as is.